xml/book_storage_admin.xml

```diff
@@ -66,11 +66,8 @@
   <xi:include href="admin_ceph_faqs.xml" parse="xml"/>
   <xi:include href="admin_ceph_troubleshooting.xml" parse="xml"/>
  </part>
- <part xml:id="admin-glossary">
-  <title>Glossary</title>
-  <xi:include href="ses_glossary.xml" parse="xml"/>
- </part>
  
+ <xi:include href="ses_glossary.xml" parse="xml"/>
```

Comment:
Move the glossary include down right before the `admin_docupdates.xml` entry please to have it positioned same as the deployment guide.

Reply:
LGTM now